static/src/stylesheets/module/facia/_container--video.scss

```diff
@@ -43,6 +43,8 @@ $video-width-desktop: 700px;
         left: 0;
         right: 0;
         bottom: 0;
+        pointer-events: none;
```

Comment:
have we made sure this isn't affecting any other containers (or is `.fc-container--video` new for this container?)